homeassistant/components/light/__init__.py

```diff
@@ -403,7 +405,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:  # noqa:
     # of the light base platform.
     hass.async_create_task(profiles.async_initialize(), eager_start=True)
 
-    def preprocess_data(data: VolDictType) -> VolDictType:
+    def preprocess_data(data: dict[str, Any]) -> VolDictType:
```

Comment:
mypy complained, and after reading through the code I think the type hint here was previously wrong.